### PR TITLE
added assay = chemistry panels

### PR DESCRIPTION
**Pull request by amapeters**

### CODE REVIEW

File: synapseAnnotations/data/experimentalData.json [hunk 1]
```diff
@@ -22,6 +22,11 @@
         "description": "An instrument or machine for weighing.",
         "source": "http://purl.obolibrary.org/obo/MMO_0000217"
       },
+      {
+        "value": "chemistry panels",
+        "description": "Groups of tests that are routinely ordered to determine a person's general health status. The tests are performed on a blood sample, usually drawn from a vein",
```
Comment (kdaily) [inline synapseAnnotations/data/experimentalData.json]:
```suggestion
        "description": "Groups of tests that are routinely ordered to determine a person's general health status. The tests are performed on a blood sample, usually drawn from a vein,",
```

**Comment by kdaily** [comment]
This also needs to be merged to the `develop` branch.

**Comment by amapeters** [comment]
This is for a panel measuring LDL, HDL, glucose, triglycerides

**Comment by allaway** [comment]
The children of the NCIT "Chemistry Test" have some good examples: https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C49237

**Comment by amapeters** [comment]
I initially considered 'Blood Chemistry Measurement' - It does have glucose and lipids.  http://purl.obolibrary.org/obo/NCIT_C47868

**Comment by allaway** [comment]
I don't forsee this being a super common assay type, so something like `blood chemistry measurement` or `comprehensive metabolic panel` might be sufficient. I assume the standard output from these procedures are not nice clean tidy datasets, but more like EHR data, right? It would probably only be worth being more specific if the data were tidied and seperated by *actual* assay.

**Comment by amapeters** [comment]
Looks good

**Pull request merged**